pubspec.yaml

```diff
@@ -27,8 +27,7 @@ dependencies:
   get_it: ^7.1.3
   provider: ^5.0.0
   uni_links: ^0.5.1
-  geocoder: ^0.2.1
-  share_plus: ^3.0.4
+  share: ^2.0.4
```

Comment:
Why did you replaced `share_plus` with `share`, because of this build is failing?

Reply:
I added it by mistake.Again I replaced it with   share_plus: ^3.0.4
.

Reply:
Please fix the conflicting files and update your commit